webpack.config.js

```diff
@@ -92,12 +98,58 @@ const createCssLoaderConfig = () =>
 
 const createCssExtractTextPlugin = () => new ExtractTextPlugin(CSS_FILENAME_OUTPUT_PATTERN);
 
+class PostCompilePlugin {
+  constructor(fn) {
+    this.fn = fn;
+  }
+
+  apply(compiler) {
+    compiler.plugin('done', (stats) => this.fn(stats));
+  }
+}
+
+const createStaticBuildPlugin = () => {
+  return new PostCompilePlugin(() => {
+    if (!BUILD_STATIC_DEMO_ASSETS || !fs.existsSync(OUT_DIR_ABS)) {
+      return;
+    }
+
+    const demosDirAbs = path.resolve('./demos');
+    const tmpDirAbs = fs.mkdtempSync(path.join(os.tmpdir(), 'mdc-web-demo-output-'));
+
+    const copyOptions = {
+      // eslint-disable-next-line no-unused-vars
+      filter: (src, dest) => {
+        return !/\.(scss|css.js)$/.test(src);
+      },
+    };
+
+    fsx.copySync(demosDirAbs, tmpDirAbs, copyOptions);
+    fsx.copySync(OUT_DIR_ABS, path.join(tmpDirAbs, DEMO_ASSET_DIR_REL), copyOptions);
+
+    if (!WRAP_CSS_IN_JS) {
+      glob.sync(path.join(tmpDirAbs, '**/*.html'))
+        .forEach((absPath) => {
+          const oldHtml = fs.readFileSync(absPath, {encoding: 'utf8'});
+          const newHtml = oldHtml.replace(/<script src="([^"]+\.css\.js[^"]*)"><\/script>/ig, (match, oldPath) => {
+            const newPath = oldPath.replace('.css.js', '.css');
+            return `<link rel="stylesheet" href="${newPath}">`;
+          });
+          fs.writeFileSync(absPath, newHtml, {encoding: 'utf8'});
+        });
+    }
+
+    fsx.removeSync(OUT_DIR_ABS);
```

Comment:
Might want to comment here or at the top as to the relationship of some of the directories to make it clear why we're deleting this folder then recreating it.

Reply:
Done